config.dot

```diff
@@ -90,7 +90,7 @@ digraph AgentEnabled {
   "Agent#log_seed_token"
   "Agent#log_error"
   "Agent#monitoring?"
-  "Agent#check_trasaction_sampler_status"
+  "Agent#check_transaction_sampler_status"
```

Comment:
I understand that this project cares about api support. If things need to be retained, I'll happily drop them. A single mention by spelling is enough as commits are tied to individual spelling family corrections.

Reply:
We've been contemplating the removal of this file (#1253). Turns out, this is one of a handful of APIs mentioned that have no reference in code. This change should be fine.